frontend/src/components/editor/output/Outputs.css

```diff
@@ -110,4 +110,5 @@
 
 .traceback-cell-link {
   color: inherit;
+  text-decoration: underline;
 }
```

Comment:
```suggestion
  text-decoration: underline;
}

.traceback-cell-link:hover {
    font-weight: 500;
}
```

Reply:
This feels weird, does is not change the width of everything?

Reply:
500 does look good actually